### ISSUE TITLE
Can't create character in last experimental - error in `anatomy.json`

### ISSUE THREAD

**Issue by mlange-42** [CleverRaven/Cataclysm-DDA#61488]
### Describe the bug

When trying to create a new character, the game returns to the main menu with this error:

```
ERROR : src/main_menu.cpp:947 [bool main_menu::new_character_tab()] Error: Json error: data/json/anatomy.json:6:5: Failed to report missing member id

    "type": "anatomy",
    "parts": [ "eyes", "head", "mouth", "torso", "arm_l", "arm_r", "hand_l", "hand_r", "leg_l", "leg_r", "foot_l", "foot_r" ]
  },
    ^
  {
    "id": "default_anatomy",
    "type": "anatomy",
```

### Steps to reproduce

* Get build [2022-10-05-1937](https://github.com/CleverRaven/Cataclysm-DDA/releases/tag/cdda-experimental-2022-10-05-1937)
* Try to create a new custom character

Build [2022-10-05-1812](https://github.com/CleverRaven/Cataclysm-DDA/releases/tag/cdda-experimental-2022-10-05-1812) works

### Expected behavior

Can create a new character without errors

### Screenshots

![grafik](https://user-images.githubusercontent.com/44003176/194172510-15b800d3-5a8f-4ee0-b801-894249820c2e.png)

### Versions and configuration

- OS: Windows 10
- Game version: Build [2022-10-05-1937](https://github.com/CleverRaven/Cataclysm-DDA/releases/tag/cdda-experimental-2022-10-05-1937)
- Graphics version: Tiles
- Ingame language: English
- Mods loaded: [ dda, no_npc_food, no_fungal_growth ]

**Comment by mqrause** [comment]
I can't reproduce this. I also don't see a missing "id" member in `anatomy.json`.

**Comment by mlange-42** [comment]
> I can't reproduce this. I also don't see a missing "id" member in `anatomy.json`.

Checked it and my `anatomy.json` looks good too. Actually, compiled the `master` corresponding to the build. Maybe should check release builds too...

**Comment by EIIKaO** [comment]
Release build `cdda-windows-tiles-x64-2022-10-05-1937` work well, can create the new chara, in my machine.

**Comment by mlange-42** [comment]
> Release build `cdda-windows-tiles-x64-2022-10-05-1937` work well, can create the new chara, in my machine.

Yeah, same here using the Launcher. Made a repo reset and `make clean`, and now recompiling (🕐) to check if it is a local problem, or something OS-specific or so.

**Comment by mlange-42** [comment]
Works now after cleanup and full recompilation.
